src/users/models/add_human_user_request.rs

```diff
@@ -0,0 +1,250 @@
+	profile: models::SetHumanProfile,
+	#[serde(rename = "email")]
+	email: models::SetHumanEmail,
+	#[serde(rename = "phone")]
+	phone: Option<models::SetHumanPhone>,
+	#[serde(rename = "metadata")]
+	metadata: Option<Vec<models::SetMetadataEntry>>,
+	#[serde(rename = "password")]
+	password: Option<models::Password>,
+	#[serde(rename = "hashedPassword")]
+	hashed_password: Option<models::HashedPassword>,
+	#[serde(rename = "idpLinks")]
+	idp_links: Option<Vec<models::IdpLink>>,
+	/// An Implementation of RFC 6238 is used, with HMAC-SHA-1 and time-step of
+	/// 30 seconds. Currently no other options are supported, and if anything
+	/// different is used the validation will fail.
+	#[serde(rename = "totpSecret")]
+	totp_secret: Option<String>,
+}
+
+impl AddHumanUserRequest {
+	pub fn new(
+		profile: models::SetHumanProfile,
+		email: models::SetHumanEmail,
+	) -> AddHumanUserRequest {
+		AddHumanUserRequest {
+			user_id: None,
+			username: None,
+			organization: None,
+			profile: profile,
+			email: email,
+			phone: None,
+			metadata: None,
+			password: None,
+			hashed_password: None,
+			idp_links: None,
+			totp_secret: None,
+		}
+	}
+
+	pub fn set_user_id(&mut self, user_id: String) {
+		self.user_id = Some(user_id);
+	}
+
+	pub fn with_user_id(mut self, user_id: String) -> AddHumanUserRequest {
+		self.user_id = Some(user_id);
+		self
+	}
+
+	pub fn user_id(&self) -> Option<&String> {
+		self.user_id.as_ref()
+	}
+
+	pub fn reset_user_id(&mut self) {
+		self.user_id = None;
+	}
+
+	pub fn set_username(&mut self, username: String) {
+		self.username = Some(username);
+	}
+
+	pub fn with_username(mut self, username: String) -> AddHumanUserRequest {
+		self.username = Some(username);
+		self
+	}
+
+	pub fn username(&self) -> Option<&String> {
+		self.username.as_ref()
+	}
+
+	pub fn reset_username(&mut self) {
+		self.username = None;
+	}
+
+	pub fn set_organization(&mut self, organization: models::Organization) {
+		self.organization = Some(organization);
+	}
+
+	pub fn with_organization(mut self, organization: models::Organization) -> AddHumanUserRequest {
+		self.organization = Some(organization);
+		self
+	}
+
+	pub fn organization(&self) -> Option<&models::Organization> {
+		self.organization.as_ref()
+	}
+
+	pub fn reset_organization(&mut self) {
+		self.organization = None;
+	}
+
+	pub fn set_profile(&mut self, profile: models::SetHumanProfile) {
+		self.profile = profile;
+	}
+
+	pub fn with_profile(mut self, profile: models::SetHumanProfile) -> AddHumanUserRequest {
+		self.profile = profile;
+		self
+	}
+
+	pub fn profile(&self) -> &models::SetHumanProfile {
+		&self.profile
+	}
+
+	pub fn set_email(&mut self, email: models::SetHumanEmail) {
+		self.email = email;
+	}
+
+	pub fn with_email(mut self, email: models::SetHumanEmail) -> AddHumanUserRequest {
+		self.email = email;
+		self
+	}
+
+	pub fn email(&self) -> &models::SetHumanEmail {
+		&self.email
+	}
+
+	pub fn set_phone(&mut self, phone: models::SetHumanPhone) {
```

Comment:
If we're providing builders we might as well take a `String` and a `bool` as separate args.